components/ProcessSteps/src/_StepHeader.scss

```diff
@@ -2,4 +2,8 @@
   display: flex;
   align-items: var(--denhaag-process-steps-step-header-align-items);
   gap: 16px;
+
+  &--nested {
+    gap: 24px;
```

Comment:
Is dit iets wat in een design token zou kunnen? Net als de overige vaste waardes.